### DISCUSSION TITLE
Downloadable Map URL

### DISCUSSION THREAD

**Discussion by n1zzo** [wavelog/wavelog#604] [Ideas]
Now that QRZ.com does not allow javascript on its iframe, it would be nice to have a URL that allows to generate a rendered version of the QSO map. So that it could still be generated on-the-fly and updated to QRZ.com.

**Comment by HB9HIL** [comment]
😊

https://github.com/wavelog/wlimager_maps

**Reply by n1zzo** [reply]
[Here is my PR](https://github.com/wavelog/wlimager_maps/pull/4). 

It is very kind of you to host this service! 😊
